com.adobe.Reader.metainfo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.adobe.Reader</id>
+
+  <name>Adobe Reader</name>
+  <summary>PDF viewer</summary>
+
+  <url type="homepage">http://www.adobe.com/products/reader.html</url>
+  <url type="help">https://helpx.adobe.com/security/products/reader-linux.html</url>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://www.adobe.com/products/eulas/pdfs/Reader_Player_AIR_WWEULA-Combined-20080204_1313.pdf</project_license>
+
+  <description>
+    <p>
+      Adobe Reader software is the free trusted standard for reliably viewing, printing, and annotating PDF documents. It's the only PDF file viewer that can open and interact with all types of PDF content, including forms and multimedia.
```

Comment:
to be honest, I'd not use the word "trusted" here.

Reply:
This blurb is from the Adobe website, but I can of course remove "trusted" if that's preferred.